### ISSUE TITLE
Apparmor denials from multipass all the time

### ISSUE THREAD

**Issue by xnox** [canonical/multipass#1552]
[multipass-denied.log](https://github.com/canonical/multipass/files/4662500/multipass-denied.log)

I expect no denials.

I'm on Focal.

**Comment by Saviq** [comment]
Hey,

What's your output from `snap connections multipass`?

**Comment by xnox** [comment]
```
$ snap connections multipass
Interface          Plug                         Slot                Notes
firewall-control   multipass:firewall-control   :firewall-control   -
home               multipass:all-home           :home               -
home               multipass:home               :home               -
kvm                multipass:kvm                :kvm                -
libvirt            multipass:libvirt            -                   -
multipass-support  multipass:multipass-support  :multipass-support  -
network            multipass:network            :network            -
network-bind       multipass:network-bind       :network-bind       -
network-control    multipass:network-control    :network-control    -
unity7             multipass:unity7             :unity7             -
wayland            multipass:wayland            :wayland            -
x11                multipass:x11                :x11                -
```

**Comment by Saviq** [comment]
OK so most of the denials are dbus to NetworkManager, or access to `/etc/ssh_config`. We can try and suppress those, but why would you "expect no denials"? Sure, in an ideal world snaps would never _try_ to get out of their designated confinement, but without being in control of 100% of the code, how would you?

Is the DENIED message itself what's bothering you? Do other snaps not cause that, or do you just hold Multipass to a higher standard?

**Comment by xnox** [comment]
CPU / memory usage. each of the those is kernel context switch and is expensive.

If the process is not allowed to do those things under snap confiment, when snap variable is set, it should not try to use them. Waste of resources.

It makes my dmesg spammy and hard to debug any other issues on my system. Although this is a red herring, it's hard to tell if it is or not. It makes me think that multipass is a broken piece of software, and leaves a bitter aftertaste that as if it tries to break into my NetworkManager and steal my netowkring credentials.

It is a UX issue to me - unnecessary scary warnings, that I as a user, can do nothing about.

**Comment by townsend2010** [comment]
Since the vast majority of the denials are about `/etc/ssh_config` and that was fixed in #2971, I'm going to close this.
